tests/client_test.py

```diff
@@ -25,11 +25,11 @@
 import grpc
 from pyfakefs.fake_filesystem_unittest import TestCase as FileTestCase
 
-import google.ads.google_ads.client
+from google.ads.google_ads import client as Client
```

Comment:
same as previous comment re: case for modules

Reply:
Same as above, I can alias if you think it's easier to read.